### ISSUE TITLE
host_details.lua returned an error

### ISSUE THREAD

**Issue by johnydoo** [ntop/ntopng#1042]
Detected Alerts > Flow Alerts > clicking on an individual host in the Description wil generate this error:

```
Script "/usr/share/ntopng/scripts/lua/host_details.lua" returned an error:

/usr/share/ntopng/scripts/lua/modules/http_lint.lua:872: [LINT] _GET["ifname"] = "tcp:__127.0.0.1:5556" parameter error: Missing validation
```

**Comment by johnydoo** [comment]
Running ntopng 2.5.170314-2398 on CentOS 7.3.1611.

**Comment by danielmeyer** [comment]
Running 2.5.170316-2509 - Pro Small Business Edition on Ubuntu 16.04.2 LTS and encountering the exact same issue.

**Comment by emanuele-f** [comment]
We are reworking alerts code right now, the fix will be available as soon as we finish it

**Comment by emanuele-f** [comment]
As discussed with @simonemainardi, this is fixed with 6d1bc0a219f3464cd4e020983309be350b53817f.
@johnydoo @danielmeyer newly generated alerts will work correctly. Please flush existing alerts if you want to start clean.
